src/Controls/src/Core/HandlerImpl/RadioButton/RadioButton.Tizen.cs

```diff
@@ -28,6 +29,9 @@ static View BuildTizenDefaultTemplate()
 				Padding = 6
 			};
 
+			Brush radioButtonCheckMarkThemeColor = ResolveThemeColor("RadioButtonCheckMarkThemeColor");
+			Brush radioButtonThemeColor = ResolveThemeColor("RadioButtonThemeColor");
```

Comment:
I feel like this code here does not make sense as it falls back to black/white instead of letting the control template make the decision. See my longer comment as well.